@here/olp-sdk-core/test/unit/NodeFileData.test.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2021 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import * as sinon from "sinon";
+import * as chai from "chai";
+import * as fs from "fs";
+import sinonChai = require("sinon-chai");
+import { NodeFileData } from "../../lib/utils/multipartupload-internal/NodeFileData";
+
+chai.use(sinonChai);
+const expect = chai.expect;
+
+describe("NodeFileData", function() {
```

Comment:
I don't see any negative tests. Can you also add tests that you do not crash or so in case invalid path is provided or not valid FileHandle.

Reply:
added.